### DISCUSSION TITLE
Me dieron de baja GitHub Copilot Education

### DISCUSSION THREAD

**Discussion by gibinaghi** [community/community#31416] [Copilot Conversations] [answered]
Tenia desde hace unos meses GitHub Copilot para educacion. Subí el certificado de alumno regular como correspondia y estuve varios meses usandolo. De un dia para el otro me dieron de baja y no tengo idea la razón. 
¿Qué puede ser? ¿Algo para hacer asi lo vuelvo a obtener?

**Comment by davecheney** [comment, marked as answer]
@gibinaghi copilot is available for free for students with an active coupon; currently `student-2020`, `student-2021` and `student-2022`. I've checked your account and you do not have one of those coupons active.

You may apply for a student developer pack here, https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student

**Comment by datacubeR** [comment]
Hi guys, I'm having the same issue. It says that my extended access has ended, but it turns out that I'm coursing my Phd for the next 4 years and for some reason my github copilot stopped working. What can I do to re enable it again, because my pro account is still working but copilot is not... Thanks in advance.

Alfonso

**Reply by davecheney** [reply]
@datacubeR I've checked your account; your student coupon is active and provides you with complementary access. Can you please tell me a bit more about what you tried, and what happened. Thank you.

**Reply by datacubeR** [reply]
In my case my copilot config says my extended access has ended but if installing the vscode extensions it does not give me the option to log in . While enabling the extension absolutely no suggestion is triggered at all.

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by datacubeR** [reply]
Not sure if something changed but Vscode is not providing inline suggestions as I code but only  when manually triggering using alt+\. Is there any way to configure it back as before?

**Reply by davecheney** [reply]
Try editing your `settions.json` file, delete the `github.copilot.enable` and `github.copilot.advanced` keys, then close and reopen the window. That will reset your local settings.

**Reply by datacubeR** [reply]
There was a inline suggestions false in the middle of my configuration. Thanks for the guidance I manage to solve it
